tox.ini

```diff
@@ -100,29 +100,36 @@ setenv =
     FALCON_ASGI_WRAP_NON_COROUTINES=Y
     FALCON_TESTING_SESSION=Y
     PYTHONASYNCIODEBUG=1
+install_command = python -m pip install --no-build-isolation {opts} {packages}
```

Comment:
I found that the `test_cython.test_imported_from_c_modules` was failing on my box. It seemed to be related to the presence of the pyproject.toml file triggering pip to enable build isolation, and therefore it wasn't finding cython in the environment. If that's true, we'll need to update our docs that talk about using cython, and we'll probably need to look into migrating wholesale over to PEP 517 at some point.

Reply:
I had something like that, but then `git clean -fdx`ed the source tree and it passed.